### DISCUSSION TITLE
Chrome settings migration help

### DISCUSSION THREAD

**Discussion by swezey** [sandboxie-plus/Sandboxie#2507] [Q&A Feedback]
First of all total newbie here....

Second of all sorry if I am in the totally wrong place...

Third pretty much a computer neophyte here....

Bottom line: please be patient.  :-)

So I "discovered" Sandboxie by googling around as I have been worried about ransomware and viruses etc and while I DO have the Microsoft Defender (or whatever it is called) on my PC, I liked the idea of running my broswer (Google Chrome always up to date) in a sandbox.  So I downloaded Sandboxie and even paid for it because I think that security is valuable but...  I can not for the life of me get my Chrome to open up in Sandboxie with all the tabs and settings intact.  It is ALWAYS like a fresh install;l;.  I have made new sandboxes, changed settings, enabled templates, mucked around with folders etc etc, and, while I am sure it is something SUPER simple, I can't make it do it for anything.  Can someone PLEASE help me figure out how to get Chrome running in Sandboxie with all my settings intact?  I don't think I am dumb but man I have spent hours and hours on this and googled it till I am blue.  Most google results are asking how NOT to have the settings preserved (which is the opposite of my problem) which leads me to believe it should be a piece of cake but I can't make it do it for love or money.  HELP!

Bill

**Comment by bastik-1001** [comment]
From what you wrote, I expect that you installed Chrome onto the real machine, which should allow you to run Chrome in a standard box (or a security enhanced box), where you have access to all the data in your profile.

To achieve that, all you should have to do is right-clicking onto Chrome, and run it by selecting "Run Sandboxed" and then selecting the box for it to run in. What happens if you do that?

Since you write that it is always like a fresh installation, the sandbox appears to be lacking reading access to the profile folder, which it should have by default, if the user Chrome is normally run as, is the same that starts it sandboxed.

**Reply by swezey** [reply]
Bastik,

  Thanks for your suggestions!  I appreciate you taking time to write an answer.  Unfortunately, I do not have an option to "Run Sandboxed" when I right-click on the Chrome icon.  Please see the screenshot below:
  
  
![image](https://user-images.githubusercontent.com/105005686/204884599-cd6315ca-64c6-45bb-82b5-77870159f453.png)

Is this a setting or something I have to enable in the Sandboxie manager?

**Reply by bastik-1001** [reply]
Sorry, I assumed it was enabled by default. You can enable it under Options > Global Settings > Shell Integration: ![shellintergration](https://user-images.githubusercontent.com/1449429/204889576-7ae5298e-dd4e-4ab4-b370-b6c7d167cb33.png)

Since I assumed the entry to be present, I considered that the easiest option, but you can run Chrome (and any other program) by right-clicking on a sandbox and then select "Run Program" to navigate to Chrome or "Run from Start Menu" then select Chrome or even "Run Default Web Browser", assuming Chrome is the default browser.

**Reply by bastik-1001** [reply]
Here you can see the run menu, after right-clicking onto a box, in Sandman, the default sandbox manager of Sandboxie Plus. ![menu with run menu_1](https://user-images.githubusercontent.com/1449429/204893005-b7048e4f-b422-4294-b6af-79c19d7b1143.png)

**Reply by bastik-1001** [reply]
And there is an [issue](https://github.com/sandboxie-plus/Sandboxie/issues/2284) with Windows 11 and the context menu in its explorer, so my suggestion wasn't the best idea to try, but my guess was that this was the quickest way to start something installed to the system.

**Reply by swezey** [reply]
Bastik,

  I am on Windows 10.  I found that the setting you mentioned WAS enabled but does not work from either the Taskbar or from using the Windows key and typing 'Chrome'.  To get it to work I had to create a shortcut for Chrome and place it on the desktop.  I then was able to use the right-click menu item to start Chrome.

  However, even after this, I found it was showing me the session from within the Sandbox NOT the session installed on my C drive.  To get this to work, I had to delete the contents of the DefaultBox so it would pick up the settings from my installed Chrome.  However, to do this I had to turn off deletion protection on the Default box.  

  This seems unnecessarily complex and I wonder how many other people have fallen into this same set of circumstances as myself.  Anyway, it is working as I expected now and I thank you for your assistance!

Bill

**Reply by isaak654** [reply]
> I am on Windows 10. I found that the setting you mentioned WAS enabled but does not work from either the Taskbar or from using the Windows key and typing 'Chrome'. 

It's because those places are not currently covered by the "Run Sandboxed" entry, which is mostly available in the Desktop context menu or in other files/folders present in Windows Explorer (also known as File Explorer). 

It could be related to the lack of [UWP](https://github.com/sandboxie-plus/Sandboxie/labels/UWP) support.

**Reply by bastik-1001** [reply]
>  However, to do this I had to turn off deletion protection on the Default box.

This shouldn't be the default setting, but yes, in order to delete the contents of a sandbox, the deletion protection has to be disabled.

Just good that it works for you, now.
